Libraries/src/Amazon.Lambda.Serialization.SystemTextJson/AwsNamingPolicy.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.Json;
+
+namespace Amazon.Lambda.Serialization.SystemTextJson
+{
+    /// <summary>
+    /// Custom AWS naming policy
+    /// </summary>
+    public class AwsNamingPolicy : JsonNamingPolicy
```

Comment:
Just brainstorming...

Could we make a custom attribute that stores the same string that JsonPropertyNameAttribute does, and read from that attribute in this class? That way, we don't need all of the macros. Optionally, we could have the string be null by default, in which case this class could read from the DataRow attribute to reduce code duplication.

Reply:
- Considering that JsonNamingPolicy is limited (you only get a string, no additional context), the solution I was suggesting could be dangerous.
- Using "DataMember" functionality dynamically is complex and messy.
- The tracking issue for having the Json serializer respect the DataMember attribute is now in the 5.0 milestone tracker https://github.com/dotnet/runtime/issues/29975

In light of these facts, this is not a viable suggestion. Approving.